Tests/NostrSDKTests/Fixtures/nip44.vectors.json

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
This file was copied from https://github.com/paulmillr/nip44/blob/b60eef26fe336e7e086b98a1bad27f0b8f09bca2/nip44.vectors.json